velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -284,21 +303,21 @@ TEST_F(HdfsFileSystemTest, missingFileViaReadFile) {
   }
 }
 
-TEST_F(HdfsFileSystemTest, schemeMatching) {
-  try {
-    auto fs = std::dynamic_pointer_cast<filesystems::HdfsFileSystem>(
-        filesystems::getFileSystem("/", nullptr));
-    FAIL() << "expected VeloxException";
-  } catch (VeloxException const& error) {
-    EXPECT_THAT(
-        error.message(),
-        testing::HasSubstr(
-            "No registered file system matched with file path '/'"));
-  }
-  auto fs = std::dynamic_pointer_cast<filesystems::HdfsFileSystem>(
-      filesystems::getFileSystem(fullDestinationPath, nullptr));
-  ASSERT_TRUE(fs->isHdfsFile(fullDestinationPath));
-}
+// TEST_F(HdfsFileSystemTest, schemeMatching) {
```

Comment:
why comment this test?

Reply:
To resolve the issue https://github.com/facebookincubator/velox/pull/9835/files#r1758102096, we move the InsertIntoHdfsTest ahead HdfsFileSystemTest.
And the InsertIntoHdfsTest class already extends HiveConnectorTestBase, which has registered the local file system [here](https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/utils/HiveConnectorTestBase.cpp#L30). Therefore, no exception should occur here.

Reply:
The behavior should not depend on the ordering of the tests. I resolved a similar issue for S3 in the past by using a different test executable. Will that help here?

Reply:
@majetideepak Yes. I put them in different test executable. Please help to review again. Thanks.